### PR TITLE
feat(context-action-menu): gh-context-action-menu

### PR DESCRIPTION
**Pull request by ffriedl89**

### CODE REVIEW

File: src/lib/context-menu/context-menu.scss [hunk 1]
```diff
@@ -0,0 +1,50 @@
+@import '../core/core';
```
Comment (thomaspink) [inline src/lib/context-menu/context-menu.scss]: don't include the whole core

File: src/lib/context-menu/README.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
+# Context Menu
+
+The Angular Groundhog Context Menu creates a menu thats hidden inside an overlay. It is possible to disable the entire context menu or the items within. Each item within is rendered as a `gh-button`. The context menu traps the focus inside the overlay until it is closed again. It sets the focus to the previously focused element when closed.
```
Comment (thomaspink) [inline src/lib/context-menu/README.md]: that's

File: src/lib/context-menu/README.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+Inherited by mixinTabIndex
+```typescript
+@Input('tabIndex')
+tabIndex: number
+```
+Gets and sets the tabIndex on the context menu, defaults to 0
+
+Inherited by mixinDisabled
+```typescript
+@Input('disabled')
```
Comment (thomaspink) [inline src/lib/context-menu/README.md]: maybe create a table instead?

File: src/lib/context-menu/context-menu-module.ts [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import {NgModule} from '@angular/core';
+import {CommonModule} from '@angular/common';
```
Comment (thomaspink) [inline src/lib/context-menu/context-menu-module.ts]: Don't mix `{stuff}` and `{ stuff }`

File: src/lib/context-menu/README.md (hunk 2 above)
Comment (ffriedl89) [reply src/lib/context-menu/README.md]: where and really?

File: src/lib/context-menu/README.md (hunk 3 above)
Comment (ffriedl89) [reply src/lib/context-menu/README.md]: markdown tables don't provide the feature to add code blocks inside

**Pull request merged**